doc/adoc/user_guide.adoc

```diff
@@ -100,7 +100,7 @@ data or processes. For example, DMS will function if /home is its own
 partition, but not if /var is on a separate partition from root.
 
 SLES Version Support::
-Upgrade is supported from SLES12 SP4 or SLES12 SP5 to SLES 15 SP1. It is
+Upgrade is supported from SLES12 SP4 or SLES12 SP5 to SLES 15 SP3. It is
```

Comment:
We should drop 12 SP4.

Reply:
Pushed up changes removing references to SLES12 SP4 as a supported migration starting point. Left in the part about SLES12 SP4 support ending in 2023.